app/models/search_builder/synthetic_category_limit.rb

```diff
@@ -0,0 +1,37 @@
+class SearchBuilder
+  # A search builder module to restrict to a certain SyntheticCollection.
+  # Won't do _anything_ just by being in the chain -- sets class_attributes
+  # on the SearchBuilder that can be used to turn it 'on', either on the class
+  # or even on an instance.
+  #
+  #.    synthetic_category_param : set to a string, and this will be taken
+  #.      from app params to identify a collection to limit to.  If unknown,
+  #.      will do nothing. (Should limit to 0 rows instead?)
+  #.    synthetic_category_force: symbol of synthetic collection key,
```

Comment:
The idea is that this module gets added to the search builder, and does _nothing_ unless it's activated. There are two ways to activate it:

1. Tell it to automatically take a param from the actual app query params, and map it to a synthetic category. 

2. Tell it to _always_ limit to a certain synthetic category. 

Both of these can be done either at the search builder _class_ level, or at the search builder _instance_ level.  The second one would be more likely at the instance level:

~~~
search_builder = # wherever you get it from
search_builder.synthetic_category_force = :portraits_and_people
search_builder.search # now when you do things with it, it's locked to portraits and people
~~~

It's the second one I actually plan to use in the "landing pages" for collections. This isn't how Hydra generally uses search builders.... but I think it probably should be, the search builder architecture is begging for it, it's very clean. 

Make sense? Maybe I need better inline docs here!